subprojects/robotpy-wpimath/tests/cpp/pyproject.toml

```
@@ -12,11 +12,17 @@ authors = [
     {name = "RobotPy Development Team", email = "robotpy@googlegroups.com"},
 ]
 license = "BSD-3-Clause"
+dependencies = [
+    "robotpy-wpimath==2025.3.2.4",
+]
```

So this doesn't seem to work with how it gets built for the test. I can back this change out if there isn't an easy fix, it is not actually required to get bazel working.

---

I would remove this change, since there's nothing to automatically update it and it doesn't provide any value. If the package fails to build that's fine.

---

There could be some merit to declaring the dependency, although I wouldn't want to have to update the version here every time. But yeah this is purely to test building against the wpimath package only, so not a big deal.
```suggestion
dependencies = [
    "robotpy-wpimath",
]
```